desktop-widgets/divelistview.cpp

```diff
@@ -402,7 +402,7 @@ void DiveListView::selectDives(const QList<int> &newDiveSelection)
 	// select the dives, highest index first - this way the oldest of the dives
 	// becomes the selected_dive that we scroll to
 	QList<int> sortedSelection = newDiveSelection;
-	qSort(sortedSelection.begin(), sortedSelection.end());
+	std::sort(sortedSelection.begin(), sortedSelection.end());
```

Comment:
Please explain: why does this here *not* make a deep copy whereas
```
	QStringList f();
	for (QString &s: f()) {
		...
	}
```
does.

I'm not yet totally convinced of these changes.

Reply:
Nevermind - in this case it *should* be a deep copy.